### PR TITLE
Loading screen

### PR DESCRIPTION
**Pull request by rukawa444**
This PR adds a loading screen.

loading

**Testing for the nth time:**

load

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-15)

**Comment by rukawa444** [comment]
Seems like I still need to write the right test for the CI pipeline to become happy.

**Comment by rukawa444** [comment]
> Go go go 😄

Finally, I fixed the cypress test! looks like I've grown so much beard from this...🐱‍👤

File: features/issues/components/issue-list/issue-row.tsx [hunk 1]
```diff
@@ -72,7 +66,7 @@ export function IssueRow({ projectLanguage, issue }: IssueRowProps) {
       </IssueCell>
       <Cell>
         <Badge color={levelColors[level]} size={BadgeSize.sm}>
-          {levelWords[level]}
+          {capitalize(level)}
```
Comment (jkettmann) [inline features/issues/components/issue-list/issue-row.tsx:69]: Why did you remove the `levelWords` again? Seemed like the right thing to do

File: features/projects/components/loading-screen/loading-screen.style.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+export const Loader = styled.div`
+  border-width: 0.5rem;
+  border-style: solid;
+  border-color: ${color("primary", 50)} ${color("primary", 50)}
+    ${color("primary", 50)} ${color("primary", 600)};
+  width: 3.625rem;
+  height: 3.625rem;
+  border-radius: 50%;
+  position: relative;
+  -webkit-animation: spin 2s infinite;
```
Comment (jkettmann) [inline features/projects/components/loading-screen/loading-screen.style.ts:19]: Is this required? Typically these kind of prefixes are added by styled-components automatically

**Comment by jkettmann** [comment]
Nice animation 🙂

File: features/issues/components/issue-list/issue-row.tsx (hunk 1 above)
Comment (rukawa444) [reply features/issues/components/issue-list/issue-row.tsx:69]: I thought it was not as It's the same with the figma design, so I changed it back to original. will fix it haha.

File: features/projects/components/loading-screen/loading-screen.style.ts (hunk 2 above)
Comment (rukawa444) [reply features/projects/components/loading-screen/loading-screen.style.ts:19]: Is the prefix that -webkit you're referring to? I'm not familiar with the jargon yet. 😅
